hibernate-reactive-core/src/test/java/org/hibernate/reactive/ReactiveSessionTest.java

```diff
@@ -700,6 +706,18 @@ public void testBatching(TestContext context) {
 		);
 	}
 
+	@Test
+	public void testBatchingConnection() {
+		Stage.Session session = getSessionFactory().openSession();
+		assertThat(((StageSessionImpl)session).getReactiveConnection()).isInstanceOf( BatchingConnection.class );
+	}
+
+	@Test
+	public void testBatchingConnectionMutiny() {
+		Mutiny.Session session = getMutinySessionFactory().openSession();
```

Comment:
You are not using the method `openMutinySession` in the superclass, meaning that the session doesn't get closed at the end of the test.